pyrule_compendium/__init__.py

```diff
@@ -70,7 +73,10 @@ def get_category(self, category: str, timeout: types.timeout=None) -> Union[dict
         if category not in ["creatures", "equipment", "materials", "monsters", "treasure"]:
             raise exceptions.NoCategoryError(category)
 
-        return self.api.request(f"/category/{category}", timeout)
+        if self.master_mode and category == "monsters":
+            return self.api.request("/category/monsters", timeout) + api_req(self.master_api.base_url, timeout)
```

Comment:
The endpoint `/master_mode/category/monsters does not exist`. It's just `master_mode` since all `master_mode` entries are monsters.

Reply:
Currently, the code doesn’t use the endpoint `/master_mode/category/monsters`. The if statement uses `/master_mode`.